container/experimental/rbe-debian8/BUILD

```diff
@@ -0,0 +1,113 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("@distroless//cacerts:cacerts.bzl", "cacerts")
+load("@jessie_package_bundle//file:packages.bzl", "packages")
+load(
+    "//skylib:packages.bzl",
+    "base_layer_packages",
+    "clang_layer_packages",
+    "java_layer_packages",
+    "python_layer_packages",
+)
+load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+    "container_layer",
+)
+
+cacerts(
+    name = "cacerts",
+    deb = packages["ca-certificates"],
+)
+
+container_image(
+    name = "fl-toolchain",
+    base = "@debian8//image",
+    layers = [
+        "base-ltl",
+        "clang-ltl",
+        "go-ltl",
+        "java-ltl",
+        "python-ltl",
+    ],
+    tags = ["manual"],
+)
+
+container_layer(
```

Comment:
as discussed offline, we should make changes upstream so that this can be a container_image target as well, but you can submit with this as it is.

Reply:
Implement compossible `container_image` in [rules_docker 335](https://github.com/bazelbuild/rules_docker/pull/335). 

As discussed, leave `container_layer`s as they are right now, and will change to `container_image` if my PR got merged.